README_cn.md

```diff
@@ -105,6 +105,10 @@ Hertz 基于[Apache License 2.0](https://github.com/cloudwego/hertz/blob/main/LI
 - 微信: CloudWeGo community
 
   ![WechatGroup](images/wechat_group_cn.png)
+## 贡献者
```

Comment:
Already have this page: at the right banner of this page

Reply:
I support adding because  people are giving themselves and their free time to contribute to open source projects in so many ways, so we believe everyone should be praised for their contributions (code or not)  and  the right banner page can only display 12

Reply:
Use the project README (or another prominent public documentation page in the project) to recognize the contributions of members of the project community.